### PR TITLE
ioctl build script(linux/mac/windows)

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: tools/ioctl/buildall.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+apt-get install gcc-multilib
+
+#windows
+GOOS=windows GOARCH=amd64   CGO_ENABLED=1  CXX_FOR_TARGET=x86_64-w64-mingw64-g++ CC_FOR_TARGET=x86_64-w64-mingw64-gcc  go build -o ioctl-windows-amd64.exe
+
+#mac
+GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 go build -o ioctl-darwin-amd64
+
+#linux
+GOOS=linux GOARCH=amd64 go build -o ioctl-linux-amd64
```
Comment (dustinxie) [inline tools/ioctl/buildall.sh:13]: 1. add a new line
2. also need linux/arm64 build

**Review by xianhuawei — COMMENTED** [review]
add linux arm64

**Comment by raullenchai** [comment]
@xianhuawei pls review this one - do we even need it?

**Comment by Liuhaai** [comment]
@dustinxie @raullenchai Should we close this PR? The build script has been implemented in https://github.com/iotexproject/iotex-core/blob/master/Makefile#L218 and https://github.com/iotexproject/iotex-core/blob/master/Makefile#L222

**Pull request closed without merging**